packages/modules/http/api.py

```diff
@@ -7,18 +7,14 @@
 log = logging.getLogger(__name__)
 
 
-def request_value(url: str) -> Optional[float]:
-    if "none" == url:
-        return None
-    else:
-        response = req.get_http_session().get(url, timeout=5)
-        response.encoding = 'utf-8'
-        log.debug("Antwort auf "+str(url)+" "+str(response.text))
-        return float(response.text.replace("\n", ""))
+def _request_value(url: str) -> float:
+    response_text = req.get_http_session().get(url, timeout=5).text
+    log.debug("Antwort auf %s: %s", url, response_text)
+    return float(response_text.replace("\n", ""))
 
 
-def create_request_function(url: str, path: str) -> Callable[[], Optional[float]]:
+def create_request_function(url: str, path: Optional[str]) -> Callable[[], float]:
     if path == "none" or path is None:
-        return lambda: 0
+        return lambda: 0.0
```

Comment:
```suggestion
        return lambda: None
```
Hier muss dann `None` zurückgegeben werden, wenn nichts konfiguriert ist.